### ISSUE TITLE
fix: `Error: Too few arguments to function Illuminate\Database\Schema\Grammars\PostgresGrammar` -- most likely a Laravel Sail connection issue?

### ISSUE THREAD

**Issue by MikeCraig418** [VemtoOrg/vemto2-issues#3]
I'm sure this is related to Laravel Sail too.

```

Log and Errors
schema-reader Error: Too few arguments to function Illuminate\Database\Schema\Grammars\PostgresGrammar::compileColumns(), 1 passed in /Users/path/vendor/laravel/framework/src/Illuminate/Database/Schema/Builder.php on line 364 and exactly 3 expected
#0 /Users/path/vendor/laravel/framework/src/Illuminate/Database/Schema/Builder.php(364): Illuminate\Database\Schema\Grammars\PostgresGrammar->compileColumns('roles')
#1 /Users/path/vendor/laravel/framework/src/Illuminate/Database/Schema/Builder.php(350): Illuminate\Database\Schema\Builder->getColumns('roles')
#2 /Users/path/vendor/laravel/framework/src/Illuminate/Database/Schema/Builder.php(261): Illuminate\Database\Schema\Builder->getColumnListing('roles')
#3 /Users/path/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(355): Illuminate\Database\Schema\Builder->hasColumn('roles', 'team_id')
#4 /Users/path/database/migrations/2023_09_25_162157_add_teams_fields.php(32): Illuminate\Support\Facades\Facade::__callStatic('hasColumn', Array)
#5 [internal function]: AddTeamsFields->up()
#6 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/MigrationDecoder.php(22): ReflectionMethod->invoke(Object(AddTeamsFields))
#7 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/index.php(77): MigrationDecoder->decode()
#8 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/common/Vemto.php(107): {closure}()
#9 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/index.php(18): Vemto::execute('schema-reader', Object(Closure))
#10 /Applications/Vemto.app/Contents/Resources/static/VMTTL1(15): require('phar:///Applica...')
#11 {main}
```

In this case, Laravel Sail (Docker) uses the following ENV config:

```
DB_CONNECTION=pgsql
DB_HOST=pgsql
DB_PORT=5432
DB_DATABASE=database
DB_USERNAME=postgres
DB_PASSWORD=password
FORWARD_DB_PORT=31621
```

When connecting to a database through Docker, the host (pgsql) is simply replaced by `host.docker.internal` (or 127.0.0.1). Secondly, the post uses `FORWARD_DB_PORT` rather than `DB_PORT`.

![](https://github.com/TiagoSilvaPereira/vemto2-issues/assets/23151962/5ffa7a27-ed74-4937-8df4-a3af8a8db6b3)

**Comment by MikeCraig418** [comment]
This is how Tinkerwell manages Docker support.

![](https://github.com/TiagoSilvaPereira/vemto2-issues/assets/23151962/85e64d03-5819-4fd7-b444-b60a51b137bb)

**Comment by christoferw** [comment]
we also need docker support

**Comment by TiagoSilvaPereira** [comment]
@christoferw @yo-mike Docker support is in the plans

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.7
